Premier league.ipynb

```diff
@@ -0,0 +1,1533 @@
+{
```

Comment:
<p>Please sort the games by date explicitly</p><p> </p><p>You sure about the -3 ? Shouldn't it be -2 ?</p><p>Be cautious about it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>